parachain/pallets/outbound-queue/src/api.rs

```diff
@@ -4,6 +4,7 @@
 
 use crate::{Config, MessageLeaves};
 use frame_support::storage::StorageStreamIter;
+use snowbridge_core::outbound::{Fees, Message, OutboundQueue, SubmitError};
 use snowbridge_outbound_queue_merkle_tree::{merkle_proof, MerkleProof};
 
 pub fn prove_message<Runtime>(leaf_index: u64) -> Option<MerkleProof>
```

Comment:
ditto, lets fix this up too.

```suggestion
pub fn prove_message<T>(leaf_index: u64) -> Option<MerkleProof>
```